packages/accordion/package.json

```diff
@@ -18,14 +18,14 @@
   "author": "Segun Adebayo <sage@adebayosegun.com>",
   "homepage": "https://github.com/chakra-ui/chakra-ui#readme",
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/types/index.d.ts",
-  "typings": "dist/types/index.d.ts",
+  "main": "dist/chakra-ui-accordion.cjs.js",
+  "module": "dist/chakra-ui-accordion.esm.js",
+  "types": "dist/chakra-ui-accordion.cjs.d.ts",
+  "typings": "dist/chakra-ui-accordion.cjs.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/cjs/index.js",
-      "default": "./dist/esm/index.js"
+      "require": "./dist/chakra-ui-accordion.cjs.js",
+      "default": "./dist/chakra-ui-accordion.esm.js"
     }
   },
```

Comment:
You'll see a similar change for each package. Since `preconstruct` build things a little differently, I had to update the references here. `preconstruct` automatically updates the `main` and `module` fields, but doesn't handle the rest of the fields (yet) so I had to build a script to automatically update the rest of them. This shouldn't ever need to be run  again, but I added it in `scripts/fix-types-and-typings.ts` for the time being just in case.